twoopstracker/settings.py

```diff
@@ -220,9 +220,19 @@
 #                  some to env vars.
 ACCOUNT_AUTHENTICATION_METHOD = "email"
 ACCOUNT_EMAIL_REQUIRED = True
-ACCOUNT_EMAIL_VERIFICATION = "none"
+ACCOUNT_EMAIL_VERIFICATION = env.str("ACCOUNT_EMAIL_VERIFICATION", "none")
 ACCOUNT_USER_MODEL_USERNAME_FIELD = None
 ACCOUNT_USERNAME_REQUIRED = False
+ACCOUNT_CONFIRM_EMAIL_ON_GET = env.bool("TWOOPSTRACKER_DEBUG", False)
+LOGIN_URL = env("LOGIN_URL")
+
+
+EMAIL_BACKEND = "django.core.mail.backends.smtp.EmailBackend"
```

Comment:
This is the [default](https://docs.djangoproject.com/en/3.2/ref/settings/#email-backend), no?